packages/terra-navigation-side-menu/src/NavigationSideMenu.jsx

```diff
@@ -223,14 +249,25 @@ class NavigationSideMenu extends Component {
     }
 
     return (
-      <ContentContainer {...customProps} header={header} fill className={sideMenuContentContainerClassNames}>
-        {this.buildListContent(currentItem)}
-      </ContentContainer>
+      <Fragment>
+        <span
```

Comment:
should we use `terra-visually-hidden-text` directly instead of copying it? we're already pulling it in the package.json

Reply:
We _should_ be using it, however it currently doesn't support ref forwarding which is needed to write the `innerText` which is needed for IE 11 support for aria-live to be read by JAWS correctly. I'm planning to log an issue to terra-core to add forward ref to visually hidden and then we can come back and update this and remove the associated styles.